Airports/EGNV/Ground Map/Geo.txt

```diff
@@ -1,254 +1,268 @@
+; Start Teeside (EGNV) Geo
```

Comment:
This line can be removed, it's auto-added by the compiler

Reply:
fixed